fMain.cs

```diff
@@ -234,44 +234,42 @@ private void bRandomise_Click(object sender, EventArgs e)
             //bgm sort
             SeedRNG();
             SortBGM();
-            if ((tROMName.Text != "") && (File.Exists(tROMName.Text)))
+            if (!((tROMName.Text != "") && (File.Exists(tROMName.Text))))
```

Comment:
Would be more readable as `if (tROMName.Text == "" || !File.Exists(tROMName.Text)`

Reply:
simplified to !File.Exists(tROMName.Text)